src/main/flight/pid.c

```diff
@@ -842,6 +840,83 @@ void FAST_CODE applySmartFeedforward(int axis)
 }
 #endif // USE_SMART_FEEDFORWARD
 
+#if defined(USE_ABSOLUTE_CONTROL)
+static void applyAbsoluteControl(const int axis, const float gyroRate,
+    const float setpointLpf, const float setpointHpf,
+    float *currentPidSetpoint, float *itermErrorRate)
+{
+    if (acGain > 0) {
+        float acCorrection = 0;
+        float acErrorRate;
+
+        const float gmaxac = setpointLpf + 2 * setpointHpf;
```

Comment:
It won't hurt to make this variables' names longer but cleaner, and also obeying to `camelCase`

Reply:
Therefore the original creater should do that, since some things still don't make sense in absolute control. This was just refactoring work